indico/core/signals/event/abstracts.py

```diff
@@ -23,3 +23,9 @@
 abstract_updated = _signals.signal('abstract-updated', '''
```

Comment:
This signal is not called anywhere - should I delete it?

Reply:
Or call it in `update_abstract`? :thinking:

Reply:
I can, but is it actually needed for anything? 😅

Reply:
dunno :sweat_smile:

Reply:
seems useful to have it, even if nothing is using it right now